src/streaming/models/CmcdModel.js

```diff
@@ -142,9 +142,20 @@ function CmcdModel() {
         streamProcessors = activeStream.getProcessors();
     }
 
+    function _applyClientDataReportingFilters(request) {
```

Comment:
I'm not sure if `_applyClientDataReportingFilters` is a good name for this method because here we are filtering and returning a boolean value, not applying something over the request that will return the request object modified. 

Also I think this method should be outside of `CmcdModel` because the `serviceLocations` and `adaptationSets` are attributes for ClientDataReporting element, that is the Parent of CMCDParameters. ClientDataReporting could have other ReportingSystem as child not just CMCDParameters. And we want  to apply the filters for all client data reporting. 

I think this method could be in HTTPLoader.js and the filter could be applied here https://github.com/qualabs/dash.js/blob/5548427475487fc647410b738045fec59711fd8b/src/streaming/net/HTTPLoader.js#L309 

What do you think?

Reply:
yes you are right, is better put the function in `HTTPLoader.js` and I think I can rename the function to `_isIncludeClientDataReportingFilters`. What do you think?

Reply:
Yes! That will work